### PR TITLE
Feature generalization

### LINKED ISSUE #72: [project][low prio] Add language mechanism documentation
Add a short section in the README file which explains what to do when a new language should be added (explaining which classes/files are language-specific).

### LINKED ISSUE #107: [code][low prio] Internationalization of features could be made more generic
The possible languages of feature names/descriptions are partly hard-coded. For the intended use of the application, that's totally fine. However, it could be made more generic.

- [x] The class `Feature` already has maps for name and description. One setter-/getter-method for name/description (instead of one for each language) would be enough if it had the language code as additional parameter.
- [x] I don't know if it's the best solution to create a localized description (e.g., containing the string *Values* or *Werte*) in `FeatureSetUtils`. An alternative would be to pass additional attributes like the statistics to the class `Feature` (using a `Map<String,String>` to store) and leaving the usage of such properties + necessary localizations to the frontend.

### PR DESCRIPTION
**Pull request by SeineEloquenz**
- [x] Add valueMap
- [x] Append values in frontend to the description to replace the old already localised description with values

close #107 
close #72

### CODE REVIEW

**Comment by SeineEloquenz** [comment]
@bendixsonnenberg @Weitspringer might one of you change the parts of the frontend displaying the features so the frontend puts together the localised part of the description where values, median, mean 1st Qu., etc. are displayed by getting the according keys from the feature?

**Comment by bendixsonnenberg** [comment]
The test are wrong, for example the feature test in line 36 is checking whether a valid value will be returned if the given language does not exist. It is expected that in the case of an unknown language the English text is returned

**Comment by bendixsonnenberg** [comment]
This will need to be handled in the API and as I won't be able to do this during the next week, someone else will have to do that.
The API call goes to httpAPI.Games::startRound().
The frontend really can't handle a redesign of the feature box, they are already packed quite close in matrix select.

**Comment by SeineEloquenz** [comment]
Feature is ready, please review

File: src/main/java/com/csselect/game/Feature.java [hunk 1]
```diff
@@ -104,34 +135,21 @@ public BufferedImage getClassGraph() {
 
     /**
      * Setter for the german name of the feature
+     * @param langCode langCode of the language to set the title for
      * @param germanName the german name
      */
-    public void setGermanName(String germanName) {
-        this.nameMap.put(GERMAN_LANGUAGE_CODE, germanName);
+    public void setName(String langCode, String germanName) {
+        this.nameMap.put(langCode, germanName);
```
Comment (Santaaa348) [inline src/main/java/com/csselect/game/Feature.java]: `142 public void setName(String langCode, String germanName) {`

It still says `germanName`. But should just be `name` if I understand correctly (appears at other sections of the code too).

**Comment by SeineEloquenz** [comment]
Wrong javadocs should be fixed now

**Pull request merged**